src/fides/api/api/v1/endpoints/privacy_request_endpoints.py

```diff
@@ -1805,80 +1790,65 @@ def get_individual_privacy_request_tasks(
 
 
 @router.post(
-    PRIVACY_REQUEST_TASK_QUEUE,
+    PRIVACY_REQUEST_REQUEUE,
     dependencies=[
         Security(verify_oauth_client, scopes=[PRIVACY_REQUEST_CALLBACK_RESUME])
     ],
-    response_model=Dict,
+    response_model=PrivacyRequestResponse,
 )
-def queue_task_manually(
+def requeue_privacy_request(
     privacy_request_id: str,
-    task_id: str,
     *,
     db: Session = Depends(deps.get_db),
-) -> Dict:
-    """
-    Manually queue a privacy request from a specific task if applicable
+) -> PrivacyRequestResponse:
     """
-    try:
-        privacy_request, request_task, _ = run_prerequisite_task_checks(
-            db, privacy_request_id, task_id
-        )
-    except (PrivacyRequestNotFound, RequestTaskNotFound) as exc:
-        raise HTTPException(
-            status_code=HTTP_404_NOT_FOUND,
-            detail=str(exc),
-        )
-    except UpstreamTasksNotReady as exc:
-        raise HTTPException(
-            status_code=HTTP_400_BAD_REQUEST,
-            detail=str(exc),
-        )
+    Endpoint for manually re-queuing a stuck Privacy Request from selected states - use with caution.
 
-    if privacy_request.status != PrivacyRequestStatus.in_processing:
-        raise HTTPException(
-            status_code=HTTP_400_BAD_REQUEST,
-            detail=f"Request failed. Cannot run {request_task.action_type.value} task {request_task.id} for privacy request {privacy_request.id} of status {privacy_request.status.value}",
-        )
+    Don't use this unless the Privacy Request is stuck.
+    """
+    pr: PrivacyRequest = get_privacy_request_or_error(db, privacy_request_id)
 
-    if request_task.status in COMPLETED_EXECUTION_LOG_STATUSES:
+    if pr.status not in [
+        PrivacyRequestStatus.pending,
```

Comment:
Ah, I should remove this status, looks to me like this can mean the request is awaiting approval.

Reply:
Ah, you're right!